MYabrv.bib

```diff
@@ -23,6 +23,7 @@ @String{COORDINATION
 @String{CPAIOR = "Proc.\ Int'l Conf.\ on Integration of Constraint Programming, Artificial Intelligence, and Operations Research (CPAIOR)"}
 @String{CP = "Proc.\ Int'l Conf.\ on Principles and Practice of Constraint Programming (CP)"}
 @String{CSMR = "Proc.\ Europ.\ Conf.\ on Software Maintenance and Reengineering (CSMR)"}
+@String{CSMR-WCRE = "Proc.\  Software Evolution Week - IEEE Conf.\ on Software Maintenance, Reengineering, and Reverse Engineering (CSMR-WCRE)"}
```

Comment:
"\" followed by two white spaces, will most likely look weird

Reply:
Is this resolved? Entry looks fine to me.